docs/dev/callstacks.qmd

```diff
@@ -0,0 +1,413 @@
+# Call stacks
```

Comment:
Maybe move all this code into a separate .jl file and `include` that here. That makes re-use easier, and also paves the way for creating CallGraph.jl in the future.